sql/ddl_internal.sql

```diff
@@ -341,6 +341,44 @@ BEGIN
 END
 $BODY$;
 
+CREATE OR REPLACE FUNCTION _timescaledb_internal.drop_chunks_type_check(
+    expected_type REGTYPE,
```

Comment:
Nit: I don't think `expected_type` is the right name -- maybe `given_type`? To me, the expected type is the one in the table, which you call actual_type (which is also a fine name)